src/shop-client.js

```diff
@@ -142,7 +142,7 @@ const ShopClient = CoreObject.extend({
    * ```
    *
    * @method fetchQuery
-   * @public
+   * @private
```

Comment:
Hm, yeah. I was originally going to go with `protected` here, but the [JSDoc ref](http://usejsdoc.org/tags-protected.html) suggested it shouldn't be used outside of a particular module. However, the [YUIdoc ref](http://yui.github.io/yuidoc/syntax/#protected) says it's for subclassing which seems a little closer to the use here. Thoughts?